darwin/dataset/download_manager.py

```diff
@@ -670,9 +674,44 @@ def _get_planned_image_paths(
             return [images_path / filename]
     else:
         for slot in annotation.slots:
+            if len(slot.source_files) > 1:
+                # Check that the item is either a DICOM series or a frame extracted from a video
+                is_dicom_series = all(
+                    source_file["file_name"].lower().endswith(".dcm")  # type: ignore
```

Comment:
is there a reason we can't or shouldn't check this with the type?

Reply:
Spoke in-person about this - Because we store `NifTI` files as type `dicom`